### PR TITLE
Made uuid changes and cleaned Uj6 vertex sdk auto ml text classification notebook 

### PR DESCRIPTION
**Pull request by bhupana**
PR problem description : Extra components. Delete that were not created in the code session were part of cleanup code. Deleted unnecessary code components from code  for Uj6 vertex sdk auto ml text classification notebook 

1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/migration/UJ6 Vertex SDK AutoML Text Classification.ipynb [hunk 1]
```diff
@@ -8,7 +8,7 @@
       },
```
Comment (andrewferlitsch) [inline notebooks/official/migration/UJ6 Vertex SDK AutoML Text Classification.ipynb:8]: <p>revert copyright back to 2021</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/904/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (andrewferlitsch) [inline notebooks/official/migration/UJ6 Vertex SDK AutoML Text Classification.ipynb:8]: <p>move dataset after objective</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/904/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (andrewferlitsch) [inline notebooks/official/migration/UJ6 Vertex SDK AutoML Text Classification.ipynb:8]: <p>use the template version for REGION</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/904/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (andrewferlitsch) [inline notebooks/official/migration/UJ6 Vertex SDK AutoML Text Classification.ipynb:8]: <p>Google Cloud Notebooks -&gt; Vertex AI Workbench Notebooks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/904/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (andrewferlitsch) [inline notebooks/official/migration/UJ6 Vertex SDK AutoML Text Classification.ipynb:8]: <p>use latest template version</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/904/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (andrewferlitsch) [inline notebooks/official/migration/UJ6 Vertex SDK AutoML Text Classification.ipynb:8]: <p>Vertex SDK&nbsp;-&gt; Vertex AI SDK</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/904/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (andrewferlitsch) [inline notebooks/official/migration/UJ6 Vertex SDK AutoML Text Classification.ipynb:8]: <p>Vertex SDK -&gt; Vertex AI SDK</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/904/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

**Comment by ivanmkc** [comment]
@bhupana Please add a proper PR description describing what this PR is for and what was done.

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**